eventlogger/src/main/resources/application.eventlogger.properties

```diff
@@ -1 +1,6 @@
-execution.env=${AB2D_EXECUTION_ENV:#{'local'}}
\ No newline at end of file
+execution.env=${AB2D_EXECUTION_ENV:#{'local'}}
+
+logger.slack.hpms.url=${AB2D_SLACK_HPMS_WEBHOOK:#{'https://slack.com'}}
+logger.slack.error.url=${AB2D_SLACK_ERROR_WEBHOOK:#{'https://slack.com'}}
+logger.slack.test.url=${AB2D_SLACK_TEST_WEBHOOK:#{'https://slack.com'}}
```

Comment:
Concerning the webhook, aren't we going to need multiple webhooks? CMS has expressed interest in us sending events to CMS slack as well as SB slack?

Reply:
You mean for the same event? I created multiple variables for different types of events and currently, they are just sending them to the same one anyway. If it comes up, it shouldn't be too hard to accommodate.